cmd/nerdctl/main_test.go

```diff
@@ -88,3 +90,58 @@ version = 2
 	base.Env = append(base.Env, "NERDCTL_TOML="+tomlPath)
 	base.Cmd("info").AssertFail()
 }
+
+// TestNerdctlSpecificConfig validates the configuration precedence [CLI, TOML, Default].
+func TestNerdctlSpecificConfig(t *testing.T) {
+	testutil.DockerIncompatible(t)
+	t.Parallel()
+	tomlPath := filepath.Join(t.TempDir(), "nerdctl.toml")
```

Comment:
use the `ncdefaults.NerdctlTOML()` method

Reply:
fixed